routes/projects.js

```diff
@@ -18,7 +18,21 @@ router.get('/:id', function(req, res, next) {
 });
 
 router.post('/', (req, res, next) => {
-  const projectData = req.body;
+  const d = new Date();
+  const lastProject = projects[projects.length - 1];
+  const idNum = lastProject.id.split('-');
+  const idIncrement = parseInt(idNum[1]) + 1;
+  const idCheck = idIncrement < 10 ? `0${idIncrement}` : idIncrement;
+  const newId = d.getFullYear() + '-' + idCheck;
```

Comment:
Need to add handling of New Year. If last project id year != current year, new project id should be currentYear-01.

Reply:
Ohhh year this is true!

Reply:
```suggestion
const lastProjectId = projects[projects.length - 1].id;
const lastProjectIdYear = lastProjectId.split('-')[0];
const lastProjectIdNum = lastProjectId.split('-')[1];

const currentYear = new Date().getFullYear().toString();
let newProjectId;
if (lastProjectIdYear === currentYear) {
  const idIncrement = parseInt(lastProjectIdNum) + 1;
  const newProjectIdNum = idIncrement < 10 ? `0${idIncrement}` : idIncrement;
  newProjectId = currentYear + '-' + newProjectIdNum;
} else {
  newProjectId = `${currentYear}-01`;
}
```